application/models/User_model.php

```diff
@@ -609,12 +613,20 @@ function users() {
 	// FUNCTION: array timezones()
 	// Returns a list of timezones
 	function timezones() {
-		$r = $this->db->query('SELECT id, name FROM timezones ORDER BY `offset`');
+		$r = $this->db->query('SELECT id, name FROM timezones');
 		$ts = array();
 		foreach ($r->result_array() as $t) {
 			$ts[$t['id']] = $t['name'];
 		}
 		return $ts;
+	// FUNCTION: string getTimezone()
+	// Returns the timezone name from the ID
+	function getTimezone($id) {
+		$query = $this->db->query('SELECT name FROM timezones WHERE id = ?', array($id));
+		if ($query->num_rows() > 0) {
+			return $query->row()->name;
+		}
+		return null;
 	}
```

Comment:
You removed a `}` by accident which causes the whole User_model to fail. 

In addition you removed the `ORDER BY offset` which causes the timezone tables in the UI to be unsorted. There is no reason to remove that.